CHANGES/7956.bugfix

```diff
@@ -0,0 +1 @@
+Removed break statement inside the 'finally' block of a try-catch statement otherwise it can suppress unhandled exceptions.
```

Comment:
FWIW a changelog entry should explain the high-level effect for the end-users. Throwing internal implementation details at them doesn't seem useful..